src/Authentication.Abstractions.Test/TestData/ArmResponseNoAzureCloud.json

```diff
@@ -0,0 +1,57 @@
+[
+  {
+    "portal": "https://portal.azure.us",
```

Comment:
I think we might want to add a specific new piece of data with only one enbtry, just for completeness.  We want to make sure we protect ourselves from future bugs where we make assumptions about this payload.

Reply:
Sure. I will add it.